csv/write.go

```diff
@@ -7,8 +7,18 @@ import (
 	"github.com/cloudquery/plugin-sdk/schema"
 )
 
-func WriteTableBatch(w io.Writer, _ *schema.Table, resources [][]any) error {
+func (cl *Client) WriteTableBatch(w io.Writer, table *schema.Table, resources [][]any) error {
```

Comment:
I looked at our file plugins again, I think we might need to change a bit the implementation here again (Sorry, didn't think about this before).

`WriteTableBatch` interface can stay the same but we will need to add a `WriteTableHeader(io.Writer, *schema.Table)` because in some scenarios like `file` we want only to write the header in the first batch so we need more control here.

Reply:
If that is the case, why not just use the original implementation with the `includeHeaders` argument?